bin/cpanorg_perl_releases

```diff
@@ -122,14 +122,18 @@ sub fetch_perl_version_data {
     my @perls;
     my @testing;
     foreach my $module ( @{ $data->{releases} } ) {
-        next unless $module->{authorized} eq 'true';
+        next unless $module->{authorized};
 
         my $version = $module->{version};
 
         $version =~ s/-(?:RC|TRIAL)\d+$//;
         $module->{version_number} = $version;
 
-        my ( $major, $minor, $iota ) = split( '[\._]', $version );
+        my ( $major, $minor, $iota ) = split( /[\._]/, $version );
+        if ( !defined $iota ) {
```

Comment:
Please add a comment documenting this case, as it's not necessarily obvious.  Maybe an example?

Reply:
Comment added